cypress/integration/userCanViewSingleRecipe.spec.js

```diff
@@ -28,7 +28,7 @@ describe('View single recipe', () => {
       status: 404
     })
     cy.get('#navbar').within(() => {
-      cy.get('#nav-listrecipes')
+      cy.get('#nav-list-recipes')
```

Comment:
I would suggest switching to name and removal of “nav”

Reply:
Alright.

Reply:
Update: all tests failed, it cannot find the name in the navbar now. Seems like it would be an easier solution to keep the id for now, it can be refactored further down the line.

Reply:
Okay